scripts/release_scripts/wrap_up_release_test.py

```diff
@@ -226,10 +240,16 @@ def mock_get_issues(unused_self, state, labels):
             if state == 'closed':
                 return [released_pr]
             else:
-                return []
+                return [pr_for_current_release]
         # pylint: enable=unused-argument
-        def mock_remove_from_labels(unused_self, unused_name):
-            check_function_calls['remove_from_labels_gets_called'] = True
+        def mock_remove_from_labels(unused_self, name):
+            if name == release_constants.LABEL_FOR_RELEASED_PRS: # pylint: disable=simplifiable-if-statement
```

Comment:
Why do you need the pylint pragma?

Reply:
See the comment here: https://github.com/oppia/oppia/pull/8952#issuecomment-606557508

Reply:
Maybe just work around it? E.g. 
```
name_of_counter_to_update = '...' if name == ... else '...'
check_function_calls[name_of_counter_to_update] = True
```

Also note that the enabled unused-argument pragma in line 244 doesn't seem to get disabled AFAICT... this is an error, right?

Reply:
Hey @seanlip sorry, missed replying on this. I tried your approach, the error still remains.

> Also note that the enabled unused-argument pragma in line 244 doesn't seem to get disabled AFAICT... this is an error, right?

Sorry, did not understand this. Can you please clarify a bit more.

Reply:
That seems weird. Why does the error occur exactly (like, what is it actually checking for)? I don't understand it in the context of my code snippet.

(We may need an extended discussion on this -- could you/@Hudda perhaps file an issue and have the discussion there?)

Re the second point: sorry, scratch that. I misread and this is an error on my part.

Reply:
Done: https://github.com/oppia/oppia/issues/8988